cabal.project

```diff
@@ -0,0 +1,6 @@
+source-repository-package
+  type: git
+  location: https://gitlab.com/leif.battermann/tinylog.git
```

Comment:
would be nice to have a link to the upstream PR here so it's easier to check whether we can bump this (not urgent).